api/queries_repo.go

```diff
@@ -514,6 +514,9 @@ func ForkRepo(client *Client, repo ghrepo.Interface, org, newName string) (*Repo
 	if newName != "" {
 		params["name"] = newName
 	}
+	if defaultBranchOnly {
+		params["default_branch_only"] = "true"
```

Comment:
`"true"` is a Go string and will serialize to JSON as a JSON string `"true"`. However, the API endpoint expects a JSON boolean. The easiest way to achieve a boolean being sent is to use a Go boolean `true`.

Reply:
I was wondering about that. I should have spent a bit more time looking through the codebase for a similar instance.